cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -192,7 +192,7 @@ func isListResource(msg *beehiveModel.Message) bool {
 		return true
 	}
 
-	if msg.Router.Operation == application.ApplicationResp {
```

Comment:
agree, in another case when watching service/endpointslice resource, reliable transmission is needed.

Reply:
I made a mistake, sorry! Because messages from dynamiccontroller for Get, List and Create method are sent by ListMessageWriteLoop, so i thought messages for Watch method should also be sent by ListMessageWriteLoop.
I found lots of objectsyncs for pods there were not scheduled to edge node were created, and i mistakenly thought there were not need.